src/explorer/SiteTreeItem.ts

```diff
@@ -124,6 +136,33 @@ export abstract class SiteTreeItem implements IAzureParentTreeItem {
             });
     }
 
+    private async enableScmDoBuildDuringDeploy(fsPath: string, runtime: string): Promise<void> {
+        const yesButton: MessageItem = { title: 'Yes' };
+        const noButton: MessageItem = { title: 'No', isCloseAffordance: true };
```

Comment:
Use 'Dont' warn again' to be consistent with our other dialogs. It also makes it more clear that this action is different than just closing the dialog

Reply:
I did 'Don't show again' since it's not really a warning